packages/2019-education/src/common/DataCollaborativeVisualizationColors.js

```diff
@@ -0,0 +1,108 @@
+import BrandColors from "./DataCollaborativeBrandColors";
```

Comment:
These should be imported from the component library rather than recreated unless we want for this package to avoid brand / visualization updates that the rest of the site receives